gptel-context.el

```diff
@@ -219,14 +237,38 @@ If PATH is a directory, recursively add all files in it.
 PATH should be readable as text."
   (interactive "fChoose file to add to context: ")
   (cond ((file-directory-p path)
-	 (gptel-context--add-directory path 'add))
-	((gptel--file-binary-p path)
+         (gptel-context--add-directory path 'add))
+        ((and gptel-context-exclude-gitignored
+              (gptel-context--skip-file-p path))
+         (message "Skipping git-ignored file: %s" path))
```

Comment:
Add to this message something like: `Unset gptel-context-exclude-gitignored to include this file`